videos/index.html

```diff
@@ -9,373 +9,22 @@
-                    </div>
-                </div>
-                <!--End of testing and refactoring screncast video-->
-
-                <!--UGtastic Interviews Sandro Mancuso video-->
-                <div class="news-v3 margin-bottom-30 bg-color-white">
-                    <img class="img-responsive full-width">
-                        <iframe style="height: 400px; width: 100%;" src="http://player.vimeo.com/video/54327320?color=ffffff"></iframe>
-                    </img>
-                    <div class="news-v3-in">
-                        <ul class="list-inline posted-info">
-                            <li>Posted Dec 2012 - Chicago, USA</li>
-                        </ul>
-                         <h2>UGtastic Interviews Sandro Mancuso</h2>
-                         <p>In this interview Codurance co-founder Sandro Mancuso talks Software Craftsmanship communities in London and Europe.</p>
-                    </div>
-                </div>
-                <!--end of UGtastic video-->
-
-                <!--Last Video-->
-                <div class="news-v3 margin-bottom-30 bg-color-white">
-                    <img class="img-responsive full-width">
-                        <iframe style="height: 400px; width: 100%;" src="http://player.vimeo.com/video/44243846"></iframe>
-                    </img>
-                    <div class="news-v3-in">
-                        <ul class="list-inline posted-info">
-                            <li>Posted 13 Jun 2012 - Agile Central Europe (ACE) - Krakow, Poland</li>
-                        </ul>
-                        <h2>Curing Agile Hangover with Craftsmanship</h2>
-                        <p>Agile adoption brings into focus the shortcomings in the existing engineering practices. After undergoing an Agile transformation for a few years at UBS, where the focus was heavy on process and light on the technical disciplines, it is an understatement to say that we still have problems. We call this the Agile Hangover. Focusing on the technical practices is essential to a successful software project, regardless of the process. Software Craftsmanship principles have been essential in tackling the problems we face.</p>
-                        <p>In this talk we will share our experiences in how we are trying to find the right balance between technical practices and processes. Empowered professional software engineers, well defined testing strategy, process automation, high investment in people development, requirements management and strong emphasis on quality have been the key areas our focus.</p>
-                    </div>
-                </div>
+                {% assign posts = site.categories.videos %}
+		{% for video in posts %}
+		<div class="news-v3 margin-bottom-30 bg-color-white">			
+			<img class="img-responsive full-width">
```

Comment:
Instead of *img* I'd suggest to use **div** with *embed-responsive* and *embed-responsive-16by9* or *embed-responsive-4by3*. This should produce responsive embedded video container. If changes is applied, fixed height can be removed from the *iframe* below.